build/gulp/tasks/stats.ts

```diff
@@ -148,7 +148,10 @@ function readCurrentBundleStats() {
 }
 
 task('stats:save', async () => {
-  const commandLineArgs = _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build'])
+  const commandLineArgs = _.pickBy(
+    _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build']),
+    val => val !== '', // ignore empty strings
```

Comment:
```suggestion
    _.identity, // ignore empty strings
```

Reply:
The reason why I haven't used `_.idenitity` is that it removes all falsy values:

```js
_.pickBy({stringVal: '', numberVal: 0, booleanVal: false}, _.identity)
{}

_.pickBy({stringVal: '', numberVal: 0, booleanVal: false}, val => val !== '')
{numberVal: 0, booleanVal: false}
```

So if you want to be able to specify `--build=0` on command line, you cannot use `_.identity`.

Reply:
👍